src/factories/WithFaceLandmarks.ts

```diff
@@ -5,64 +5,130 @@ import { isWithFaceDetection, WithFaceDetection } from './WithFaceDetection';
+  const calcRoll = (lever, pivot) => {
+    // When rolling, the head seems to pivot from the nose/lips/chin area.
+    // So, we'll choose any two points from the facial midline, where the first point should be the pivot, and the other "lever"
+    // Plan/Execution: get the hypotenuse & opposite sides of a 90deg triangle ==> Calculate angle in radians
+
+    const hypotenuse = Math.hypot(pivot._x - lever._x, pivot._y - lever._y);
+    const opposite = pivot._y - lever._y;
+    const angleInRadians = Math.asin(opposite / hypotenuse);
+    const angleInDegrees = degrees(angleInRadians);
+    const normalizeAngle = Math.floor(90 - angleInDegrees);
+    // If lever more to the left of the pivot, then we're tilting left
+    // "-" is negative direction. "+", or absence of a sign is positive direction
+    const tiltDirection = pivot._x - lever._x < 0 ? -1 : 1;
+    const result = normalizeAngle * tiltDirection;
+    return result;
+  };
+
+  const calcPitch = (leftPoint, midPoint, rightPoint) => {
+    // Theory: While pitching, the nose is the most salient point --> That's what we'll use to make a trianle.
+    // The "base" is between point that don't move when we pitch our head (i.e. an imaginary line running ear to ear through the nose).
+    // Executuin: Get the opposite & adjacent lengths of the triangle from the ear's perspective. Use it to get angle.
+
+    const base = calcLengthBetweenTwoPoints(leftPoint, rightPoint);
+    // adjecent is base/2 technically.
+    const baseCoords = {
+      _x: (leftPoint._x + rightPoint._x) / 2,
+      _y: (leftPoint._y + rightPoint._y) / 2,
+    };
+    const midToBaseLength = calcLengthBetweenTwoPoints(midPoint, baseCoords);
+    const angleInRadians = Math.atan(midToBaseLength / base);
+    const angleInDegrees = Math.floor(degrees(angleInRadians));
+    // Account for directionality.
+    // pitch forwards (_i.e. tilting your head forwards) is positive (or no sign); backward is negative.
+    const direction = baseCoords._y - midPoint._y < 0 ? -1 : 1;
+    const result = angleInDegrees * direction;
+    return result;
+  };
+
+  angle.roll = calcRoll(pt[27], pt[66]);
+  angle.pitch = calcPitch(pt[14], pt[30], pt[2]);
+  angle.yaw = calcYaw(pt[14], pt[33], pt[2]);
 
   return angle;
 }
 
-export function extendWithFaceLandmarks<TSource extends WithFaceDetection<{}>, TFaceLandmarks extends FaceLandmarks = FaceLandmarks68 >(sourceObj: TSource, unshiftedLandmarks: TFaceLandmarks): WithFaceLandmarks<TSource, TFaceLandmarks> {
+export function extendWithFaceLandmarks<
+  TSource extends WithFaceDetection<{}>,
+  TFaceLandmarks extends FaceLandmarks = FaceLandmarks68
+>(
+  sourceObj: TSource,
+  unshiftedLandmarks: TFaceLandmarks,
+): WithFaceLandmarks<TSource, TFaceLandmarks> {
   const { box: shift } = sourceObj.detection;
-  const landmarks = unshiftedLandmarks.shiftBy<TFaceLandmarks>(shift.x, shift.y);
+  const landmarks = unshiftedLandmarks.shiftBy<TFaceLandmarks>(
```

Comment:
unnecessary change.

Reply:
Same response as above: change made to remove lint error(s)